src/shapes/textbox.class.js

```diff
@@ -353,10 +376,6 @@
         infixWidth = splitByGrapheme ? 0 : this._measureWord([infix], lineIndex, offset);
         offset++;
```

Comment:
> why here is lenght + 1 and before was jsut lenght?

Here offset is incremented. I thought it's because of the space between worsds.
That's why I thought I should increment as well. But tests fail.

Reply:
Resolved